### PR TITLE
How to write cutom Jekyll syntax

### PR DESCRIPTION
**Pull request by agude**
Writing up the work done in https://github.com/MiniFate/MiniFate/pull/43 .

### CODE REVIEW

File: _drafts/2021-02-08-custom_markdown_for_github_pages.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  osciliscope to adjust a circuit board on a Stansaab 'Censor' computer at
+  Standard Radio's factory in Ulvsunda, Stockholm.
+categories:
+  - software-development
+---
+
+{% capture file_dir %}/files/jekyll/{% endcapture %}
+
+[Markdown] is a great way to write; simple enough to be read as text, with all
+the power of HTML. I have been using it for my own notes for a decade and this
```
Comment (charles-uno) [inline _drafts/2021-02-08-custom_markdown_for_github_pages.md:19]: "all the power of HTML" isn't really true, but also that's sorta the point? HTML can do anything, but Markdown only does the stuff you care about

File: _drafts/2021-02-08-custom_markdown_for_github_pages.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+[Liquid]: https://shopify.github.io/liquid/
+[Github Pages]: https://pages.github.com/
+
+## Layouts
+
+The [order of interpretation][ooi] for Jekyll runs Liquid _before_ compiling
+Markdown to HTML, which means you can't modify the HTML with Liquid. But
+pushing content to a layout happens _after_ the HTML is compiled, and layouts
+can contain their own Liquid code. This allows us to run Liquid on the full
+HTML of the page!
```
Comment (charles-uno) [inline _drafts/2021-02-08-custom_markdown_for_github_pages.md:53]: You can't run Liquid on the HTML, but luckily you can run liquid on the HTML! I think this paragraph could use one more sentence in the middle to clarify

File: _drafts/2021-02-08-custom_markdown_for_github_pages.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+%}
+```
+{% endraw %}
+
+Which can be fully customized with CSS.
+
+This method has two limitations:
+
+- We have to use characters that the Markdown parser won't interpreted, so `_`
+  and `*` won't work.
```
Comment (charles-uno) [inline _drafts/2021-02-08-custom_markdown_for_github_pages.md:130]: Worth also mentioning backticks and tildes?

**Review by charles-uno — APPROVED** [review]

File: _drafts/2021-02-08-custom_markdown_for_github_pages.md (hunk 1 above)
Comment (agude) [reply _drafts/2021-02-08-custom_markdown_for_github_pages.md:19]: It actually is true! Markdown is a strict superset of HTML. 😊

But I think I get your point.

**Comment by agude** [comment]
Already deployed: https://github.com/agude/agude.github.io/commit/51b7134cf680ae60bb3f42c3148e1fcd8635e566

**Pull request closed without merging**